service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -103,9 +103,6 @@ protected boolean doRegister() {
         return false;
       }
 
-      // In re-register microservice case, the old instanceId should not be cached
-      microservice.getInstance().setInstanceId(null);
-
       LOGGER.info(
```

Comment:
Maybe we need to set instance id to null when service id is empty. (When we delete all information in service center, both the service and instance is a new one.)

Reply:
we can think of a process to be an instance, use the old instance id is reasonable